spec/game_spec.rb

```diff
@@ -1,53 +1,108 @@
-require './lib/game'
-require './lib/player'
+require './lib/game.rb'
+require './lib/player.rb'
 
-RSpec.describe 'Game methods' do
-  let(:player1) { Player.new('Lamia', 'x') }
-  let(:player2) { Player.new('Mike', 'o') }
+RSpec.describe 'testing the game methods' do
+  let(:player1) { Player.new('Grace', 'x') }
+  let(:player2) { Player.new('Olamide', 'o') }
   let(:game) { Game.new(player1, player2) }
 
-  it 'should return an array of x and o' do
-    expect(Game.choose_signs.sort).to eql(%w[x o].sort)
+  it 'it chooses sign for players' do
+    arr = Game.choose_signs
+    expect(arr.sort).to eq(%w[x o].sort)
   end
 
-  context 'Validate the moves' do
-    it 'should check for the numbers between 1 to 9' do
-      expect(game.validate_move(3)).to eql(3)
+  it 'it chooses sign for players' do
+    arr = Game.choose_signs
+    expect(arr.sort).not_to eq(%w[x k].sort)
+  end
+
+  context 'validate the players move' do
+    it 'should check the moves between 1 to 9' do
+      expect(game.validate_move(6)).not_to eq(9)
     end
 
-    it 'should return -2 for string inputs' do
-      # p game.validate_move('abc')
-      expect(game.validate_move('abc')).to eql(-2)
+    it 'should check the moves between 1 to 9' do
+      expect(game.validate_move(6)).to eq(6)
     end
 
-    it 'will check for unique numbers' do
-      game.choosed_nums = [5]
-      expect(game.validate_move(5)).to eql(-1)
+    it 'should check if number is invalid' do
+      expect(game.validate_move('p')).to eq(-2)
     end
+
+    it 'should check if number is invalid' do
+      expect(game.validate_move('y')).not_to eq(-5)
+    end
+  end
+
+  it 'validates player name if greater than 3' do
+    expect(Game.validate_name('Grace')).to be(true)
+  end
+
+  it 'validates player name if greater than 3' do
+    expect(Game.validate_name('Grace')).not_to be(false)
+  end
+
+  it 'validates player name if not less than 3' do
+    expect(Game.validate_name('ab')).to be(false)
   end
 
-  it 'should return true if input length is grater than 2' do
-    expect(Game.validate_name('Lamia')).to be true
+  it 'validates player name if not less than 3' do
+    expect(Game.validate_name('ab')).not_to be(true)
   end
 
-  it 'should return array of players' do
-    expect(game.getcurrent_player).to eql([player1, player2])
+  it 'get the current player' do
+    expect(player1.name).to eq('Grace')
+    expect(player1.sign).to eq('x')
```

Comment:
You should have one `it` block for each expected behaviour. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/Ruby_testing/itblock.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59566098/))